datasets/librispeech_asr/librispeech_asr.py

```diff
@@ -159,6 +271,9 @@ def _generate_examples(self, files):
                         id_, transcript = line.split(" ", 1)
                         audio_file = f"{id_}.flac"
                         speaker_id, chapter_id = [int(el) for el in id_.split("-")[:2]]
+                        audio_file = (
+                            os.path.join(local_extracted_archive, audio_file) if local_extracted_archive else None
```

Comment:
@lhoestq is the whole speech data now stored twice in non-streaming mode? Once as the original flac and once as an array of bytes? 

Should we change line 266: from `audio_data[id_] = f.read()` to just `audio_data[id_] = f` and change the line: `audio = {"path": transcript["file"], "bytes": audio_data[transcript["id"]]}` at the moment it looks to me like both the bytes and the file is saved in non-streaming mode which would create a huge overhead in memory no?

Reply:
Opening a new PR for this to change it internally in `src/datasets` ...

Reply:
Will be fixed in https://github.com/huggingface/datasets/pull/4187 :)